nuxeo-core/nuxeo-core-management/src/main/java/org/nuxeo/ecm/core/management/api/ProbeManager.java

```diff
@@ -36,5 +39,34 @@
     ProbeInfo getProbeInfo(String name);
 
     ProbeInfo getProbeInfo(Class<? extends Probe> probeClass);
-
+    
+    /**
+     * List of probes evaluated for a health check
+     * @since 9.3
+     */
+    Collection<ProbeInfo> getAllContributeToHealthCheckProbeInfos();
```

Comment:
I don't understand the naming. Do you mean `getAllContributionsToHealthCheckProbeInfos` ? That seems quite long.